Resources/Maps/saltern.yml

```diff
@@ -47709,4 +47668,62 @@ entities:
     pos: 4.5,-30.5
     parent: 853
     type: Transform
+- uid: 4881
+  type: AirlockMaintBarLocked
+  components:
+  - pos: -12.5,-8.5
+    parent: 853
+    type: Transform
+  - containers:
+      board: !type:Container
+        ents: []
+    type: ContainerContainer
+- uid: 4882
+  type: WeldingFuelTankFull
+  components:
+  - pos: -15.5,-12.5
+    parent: 853
+    type: Transform
+- uid: 4883
+  type: solid_wall
+  components:
+  - pos: -13.5,-9.5
+    parent: 853
+    type: Transform
+- uid: 4884
+  type: AirlockMaint
+  components:
+  - pos: -14.5,-9.5
+    parent: 853
+    type: Transform
+  - containers:
+      board: !type:Container
+        ents: []
+    type: ContainerContainer
```

Comment:
@Zumorica these aren't supposed to be here right?

Reply:
It's fine as there's not an entity inside it